README.md

```diff
@@ -53,6 +53,8 @@ This will install `http-server` globally so that it may be run from the command
 
 `-P` or `--proxy` Proxies all requests which can't be resolved locally to the given url. e.g.: -P http://someurl.com
 
+`--proxyOptions` Pass proxy [options](https://github.com/nodejitsu/node-http-proxy#options) using nested dotted objects. e.g.: --proxyOptions.secure false
```

Comment:
We will soon be moving to using nconf for CLI option parsing (see #131), so I don't know whether this way of passing options will still work. That needs to be investigated, possibly after #131 merges.